src/ingest_validation_tests/segmentation_mask_imagesize_validation.py

```diff
@@ -0,0 +1,90 @@
+from pathlib import Path
+from typing import List, Optional
+
+import tifffile
+import xmlschema
+from ingest_validation_tools.plugin_validator import Validator
+from utils import GetParentData
+
+
+def get_ometiff_size(file) -> Optional[str, dict]:
```

Comment:
Should be Union, not Optional (I think this is why the build is failing in Travis CI actually).